packages/x-codemod/README.md

```diff
@@ -170,3 +171,30 @@ If you were always using the same text value in all your components, consider mo
 ```
 
 You can find more details about this breaking change in [the migration guide](https://next.mui.com/x/migration/migration-pickers-v5/#rename-the-locale-prop-on-localizationprovider).
+
+#### `column-menu-components-rename`
```

Comment:
I am thinking to categorize codemods related to grid and pickers so that users using one of the products are more convenient.
At least we can consider segregating them in documentation if not in folder structure too.
Though it's not a big deal, but we can also think about dividing `preset-safe` into `preset-safe-grid` and `preset-safe-pickers` to make things more specific.

Reply:
We discussed it in https://github.com/mui/mui-x/pull/7337#discussion_r1058855910

I'm in favor of also splitting the folder structure.

Reply:
> Though it's not a big deal, but we can also think about dividing `preset-safe` into `preset-safe-grid` and `preset-safe-pickers` to make things more specific.

Yes. We agreed on it.
So feel free to either adjust it with this PR introducing 1st data-grid codemod.
Or I could adjust the structure after this codemod is merged. 😉

Reply:
Done, please check if you see anything missing

Reply:
I'd suggest updating the data grid migration doc page to be more in line with picker:
- mention running codemods
- mark breaking changes handled by codemod